tests/rhtap-demo/config/scenarios.go

```diff
@@ -201,6 +201,71 @@ var TestScenarios = []TestSpec{
 			},
 		},
 	},
+	{
+		Name:            "DEVHAS-337: creates quarkus application from a private repository which contain a devfile",
+		ApplicationName: "private-devfile",
+		Components: []ComponentSpec{
+			{
+				Private:           true,
+				Name:              "quarkus-devfile",
+				ContainerSource:   "",
+				Language:          "Java",
+				GitSourceUrl:      "https://github.com/redhat-appstudio-qe/private-quarkus-devfile-sample.git",
+				GitSourceRevision: "",
+				GitSourceContext:  "",
+				HealthEndpoint:    "/hello-resteasy",
+			},
+		},
+	},
+	{
+		Name:            "DEVHAS-337: creates golang application from a private repository which contain a devfile",
+		ApplicationName: "private-devfile",
+		Components: []ComponentSpec{
+			{
+				Private:        true,
+				Name:           "go-devfile-private",
+				Language:       "Go",
+				GitSourceUrl:   "https://github.com/redhat-appstudio-qe/devfile-sample-go-basic-private-dockerfile-full-private.git",
```

Comment:
The Dockerfile URI is https://raw.githubusercontent.com/maysunfaisal/devfile-sample-go-basic-private/main/docker/Dockerfile, will this work with QE's token?